ui/admin/app/instance-initializers/feature-edition.js

```diff
@@ -1,6 +1,24 @@
-export function initialize(owner) {
+import fetch from 'fetch';
+
+export async function initialize(owner) {
   const service = owner.lookup('service:feature-edition');
-  service.initialize();
+  const url = '/metadata.json';
+  let edition;
+  let features;
+
+  // Attempt to load edition for the metadata JSON
+  try {
+    const response = await fetch(url);
+    const json = await response.json();
+    edition = edition || json?.license?.edition;
+    features = json?.license?.features;
+  } catch (e) {
+    // if the request fails, the service will initialize the default edition
```

Comment:
None blocking comment: In case of failure, do we want the user know?

Reply:
The metadata endpoint isn't required and should be allowed to error.  When that happens we default to the OSS edition.

Reply:
I understood that part. My question is: Would the user be informed? or because the endpoint isn't required (as you mention) the user does not need to know?